src/components/TextInput/index.jsx

```diff
@@ -19,8 +19,10 @@ const TextInput = React.forwardRef(
   (
     {
       label,
+      isLabelBold,
```

Comment:
Note: the ticket itself calls for this prop to be enabled "by default". Most text inputs with labels in the platform I've seen do not have bold treatment by default. I'm not really comfortable adding this right now unless we're 100% sure that we're not going to get any developer / product / design pushback on this.

I think we're also unclear on when designers would choose to bold or not bold something, which sort of is another data point that we're not really sure if this should be enabled by default, and it's certainly not used a lot currently, so again I'm just wary of making this a default.